### ISSUE TITLE
No Atmosphere Planet

### ISSUE THREAD

**Issue by misternika77** [Eggs-D-Studios/wos-issues#245]
### Guidelines

- [X] I checked for duplicate suggestions
- [X] My suggestion intends to add or improve something

### Version

Development (Staging)

### Topic

A game mechanic (e.g. a part, a feature, etc)

### Why are you making your suggestion?

Right now, there should be non atmosphere planets to make stuff quite challenging.

### What do you propose? What should change, or what should be added?

Basically any rocky planet without any oxygen rich atmosphere or actual atmosphere itself, they will have many varities of resources even the rare ones, thoe the player must have a ship or base with cosmic radiation protection and any machinery that gives oxygen.

**Comment by idfreud** [comment]
Those planets already exist. Use a telescope to verify that "breathable atmosphere" is "false". These planets must be barren, desert or exotic.

**Comment by misternika77** [comment]
i literally forgor

**Comment by hail12pink** [comment]
smh...
